pkg/resources/database.go

```diff
@@ -82,9 +128,37 @@ func CreateDatabase(d *schema.ResourceData, meta interface{}) error {
 		return createDatabaseFromReplica(d, meta)
 	}
 
+	if _, ok := d.GetOk("from_replica_config"); ok {
+		return createDatabaseFromReplicaConfig(d, meta)
+	}
+
+	// If set, verify parameters are valid and attempt to enable replication
+	if v, ok := d.GetOk("replication_configuration"); ok {
+		replicationConfiguration := v.([]interface{})[0].(map[string]interface{})
+		ignoreEditionCheck := replicationConfiguration["ignore_edition_check"].(bool)
+
+		if ignoreEditionCheck == false {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>S1002: should omit comparison to bool constant, can be simplified to `!ignoreEditionCheck` (gosimple)